scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -166,13 +166,13 @@
                      'many times. The purpose of this flag is to make it '
                      'possible to write regression tests that are not '
                      'bottlenecked by CNS throughput.')
-flags.DEFINE_string('local_parameter_device', 'gpu',
+flags.DEFINE_enum('local_parameter_device', 'gpu', ('cpu', 'gpu'),
                     'Device to use as parameter server: cpu or gpu. For '
                     'distributed training, it can affect where caching of '
                     'variables happens.')
-flags.DEFINE_string('device', 'gpu',
+flags.DEFINE_enum('device', 'gpu', ('cpu', 'gpu'),
```

Comment:
Either add the capitalized 'CPU' and 'GPU' strings to this list, or change benchmark_cnn_test.py to change all instances of 'CPU' to 'cpu' and 'GPU' to 'gpu'